libnd4j/blas/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set( CMAKE_VERBOSE_MAKEFILE on)
+set(CMAKE_VERBOSE_MAKEFILE OFF)
```

Comment:
same as above

Reply:
@sshepel is agreed, I asked him

Reply:
Oh, sorry, I though @shyrma was talking about merge... Well it would be better for us to keep build logs smaller, unless you really need those debug messages...